mobile/lib/modules/home/ui/asset_grid/thumbnail_placeholder.dart

```diff
@@ -0,0 +1,41 @@
+import 'package:flutter/material.dart';
+
+class ThumbnailPlaceholder extends StatelessWidget {
+  final EdgeInsets margin;
+  final double width;
+  final double height;
+
+  const ThumbnailPlaceholder({
+    super.key,
+    this.margin = EdgeInsets.zero,
+    this.width = 250,
+    this.height = 250,
+  });
+
+  static const _brightColors = [
+    Color(0xFFF1F3F4),
+    Color(0xFFB4B6B8),
+  ];
+
+  static const _darkColors = [
+    Color(0xFF3B3F42),
+    Color(0xFF2B2F32),
+  ];
+
+  @override
+  Widget build(BuildContext context) {
+    final brightness = MediaQuery.platformBrightnessOf(context);
+    return Container(
+      width: width,
+      height: height,
+      margin: margin,
+      decoration: BoxDecoration(
+        gradient: LinearGradient(
+          colors: brightness == Brightness.light ? _brightColors : _darkColors,
+          begin: Alignment.topCenter,
```

Comment:
Since we have a theme switcher from within the app, the apps theme need not be in sync with the platforms theme. Kindly use the isDarkTheme getter from the context itself, which checks the brightness from the current themedata of the context. Since it is already a boolean, we need not explicitly check for equality when assigning colors to the gradient as well

Reply:
Thank you, I've addressed this in my revision.